lib/maintainers.nix

```diff
@@ -48,4 +48,10 @@
     githubId = 39652808;
     name = "Kareem Medhat";
   };
+  svl = {
+    matrix = "@gyn:matrix.org";
+    github = "4lxs";
+    githubId = 66408983;
+    name = "svl";
+  };
```

Comment:
Create a separate commit for adding you as a maintainer.
In the end,  you should have two commits:
- `helpers/maintainers: add svl`
- `plugins/competitest: init`